src/rpc/blockchain.cpp

```diff
@@ -808,6 +808,24 @@ UniValue gettxoutsetinfo(const UniValue &params, bool fHelp)
     return ret;
 }
 
+UniValue evicttransaction(const UniValue &params, bool fHelp)
+{
+    if (fHelp || params.size() < 1)
```

Comment:
should probably add a simple test for this in one of the mempool_* python scripts.

Reply:
Seconded that it would be nice, but i don't consider not having one a reason to block the merge